cli/cpm/Cargo.toml

```diff
@@ -4,10 +4,20 @@ version = "0.1.0"
 edition = "2021"
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-cw-code-id-registry = { version = "*", path = "../../contracts/cw-code-id-registry" }
+cw-code-id-registry = { version = "*", path = "../../contracts/cw-code-id-registry", features = ["library"] }
+
+cosm-orc = { version = "2.6.0", features = ["optimize", "chain-reg"] }
```

Comment:
Just a warning that adding the `optimize` feature will increase build times a lot, since we are pulling in cw-optimizoor.

Reply:
Yeah Im just going to add an `optimize` feature that is off by default, since it adds like ~8 mins to the build to pull in cw-optimizoor.